components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -2,6 +2,9 @@ import { Template } from "./template";
 
 /**
  * Breadcrumbs show hierarchy and navigational context for a user's location within an app.
+ * 
+ * Note that any breadcrumb item can be disabled by adding the `is-disabled` class name
+ * or utilizing the `isDisabled: true` parameter.
```

Comment:
I intentionally left the `Disabled` breadcrumb story off, in favor of adding a note here instead. It just didn't feel like that was an actual "story." It wasn't like the entire breadcrumb component was disabled- it's that a single item could be disabled. 

Again, open to thoughts on this if anybody feels strongly.